cfgov/unprocessed/js/apps/financial-well-being/fwb-questions.js

```diff
@@ -1,83 +1,169 @@
-/* eslint no-global-assign: "off" */
 'use strict';
 
-var Analytics = require( '../../modules/Analytics' );
+const Analytics = require( '../../modules/Analytics' );
 
+/**
+ * Initialize the questionnaire
+ */
 function init() {
+  const questionElems = document.querySelectorAll( '#quiz-form fieldset' );
```

Comment:
For clarity because there was also `questionStates`. I actually noticed today that we use `thingDom` everywhere else, so I'm going to update to match that.

Reply:
I'm not a fan of that convention, but if it's what we're already doing...